Client/Frontend/BraveVPN/EnableVPNSettingHeader.swift

```diff
@@ -91,13 +91,20 @@ class EnableVPNSettingHeader: UIView {
         }
         
         contentView.snp.makeConstraints {
-            $0.top.leading.trailing.equalToSuperview().inset(8)
+            $0.top.equalToSuperview().inset(8)
+            if #available(iOS 14, *) {
```

Comment:
Inset grouped style is available on iOS 13 as well, so this should be 13 instead of 14